firestore/src/include/firebase/firestore.h

```diff
@@ -264,8 +264,16 @@ class Firestore {
    * transaction is valid as long as the Future has not yet completed.)
    *
    * @return A Future that will be resolved when the transaction finishes.
+   *
+   * @note This method is only available when using the STLPort C++ runtime
+   * library.
+   *
+   * @deprecated STLPort support in Firestore is deprecated and will be removed
```

Comment:
I'd rephrase the comment a little bit, particularly to make it clear that the STLPort support was already removed. Consider something like:
```
@deprecated This method was originally added to support the STLPort C++ runtime on Android.
Firebase support for STLPort has been removed, and consequently this method is
deprecated and will be removed in a future release.
```

Reply:
Done